xbmc/Application.cpp

```diff
@@ -4864,6 +4865,11 @@ std::string CApplication::GetCurrentPlayer()
   return m_pPlayer->GetCurrentPlayer();
 }
 
+std::shared_ptr<CApplicationPlayer> CApplication::GetAppPlayer()
+{
+  return m_pPlayer;
```

Comment:
m_pPlayer is a unique_ptr. not sure if passing it away as shared_ptr make sense.